### PR TITLE
File id changes

### PR DESCRIPTION
**Pull request by EmilyMcCarthy**

### CODE REVIEW

**Comment by zanchin** [comment]
do docs need to be updated to reflect any of these changes?

**Comment by EmilyMcCarthy** [comment]
No, docs already reflect the fileId changes

File: src/main/java/com/cloudmine/api/CMApiCredentials.java [hunk 1]
```diff
@@ -47,11 +48,14 @@ public static synchronized CMApiCredentials initialize(String id, String apiKey,
                 throw new CreationException("Running on android and application context not provided, try passing getApplicationContext to this method");
             }
 
-            Class.forName("com.cloudmine.api.DeviceIdentifier").getMethod("initialize", contextClass).invoke(null, context);
+//            Class.forName("com.cloudmine.api.DeviceIdentifier").getMethod("initialize", contextClass).invoke(null, context);
```
Comment (dmansen) [inline src/main/java/com/cloudmine/api/CMApiCredentials.java]: accidental comment-out?

Comment (EmilyMcCarthy) [inline src/main/java/com/cloudmine/api/CMApiCredentials.java]: No - for some reason, on 2.2.2 getMethod returns null, so I replaced it by
getting all of the methods and iterating through them until I find
initialize

Comment (dmansen) [inline src/main/java/com/cloudmine/api/CMApiCredentials.java]: ah, got it. we should take out the commented line regardless though

Comment (EmilyMcCarthy) [inline src/main/java/com/cloudmine/api/CMApiCredentials.java]: Done

**Pull request closed without merging**